views/homepage/SubmitFormSection/Form.tsx

```diff
@@ -0,0 +1,176 @@
+"use client"
+import React, { useEffect, useState } from "react"
+import { useForm, Controller, SubmitHandler } from "react-hook-form"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons"
+
+interface FormData {
+  email: string
+  caseTitle: string
+  caseStory: string
+}
+
+export function CaseForm() {
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+    reset,
+    watch,
+    trigger,
+  } = useForm<FormData>()
+  const [emailFieldInteracted, setEmailFieldInteracted] = useState(false)
+  const [submissionSuccess, setSubmissionSuccess] = useState(false)
+
+  useEffect(() => {
+    trigger("email")
+  }, [trigger, watch("email")])
+
+  const onSubmit: SubmitHandler<FormData> = (data) => {
+    const newSubmission = {
+      id: new Date().getTime(),
+      ...data,
+    }
+    const existingSubmissions: any[] = JSON.parse(
+      localStorage.getItem("submissions") || "[]"
+    )
+
+    existingSubmissions.push(newSubmission)
+
+    localStorage.setItem("submissions", JSON.stringify(existingSubmissions))
+
+    setEmailFieldInteracted(false)
+    setSubmissionSuccess(true)
+
+    reset({}, { keepValues: false })
+    setTimeout(() => {
+      setSubmissionSuccess(false)
+    }, 5000)
+    const emailInput = document.querySelector(
+      'input[name="email"]'
+    ) as HTMLInputElement
+    const caseTitleInput = document.querySelector(
+      'input[name="caseTitle"]'
+    ) as HTMLInputElement
+    const caseStoryTextarea = document.querySelector(
+      'textarea[name="caseStory"]'
+    ) as HTMLTextAreaElement
+
+    if (emailInput && caseTitleInput && caseStoryTextarea) {
+      emailInput.value = ""
+      caseTitleInput.value = ""
+      caseStoryTextarea.value = ""
+    }
```

Comment:
This looks hacky to reset form values and I think this can all be done by `reset` method from react-hook-form